### PR TITLE
17519 - Indicate to user that cancelled/expired NRs will not be returned by NR search

### PR DESCRIPTION
**Pull request by chenhongjing**
*Issue #:*
https://github.com/bcgov/entity/issues/17519

*Description of changes:*
on My Business Registry Dashboard
- Included the word 'active' in initial text
- Changed no matches state text for NR search
- app version = 2.4.26

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/package-lock.json
Comment (severinbeauvais) [inline auth-web/package-lock.json:8134]: Hongjing, what version of Node are you using?

(This is an inconsequential change, but it looks like your "npm i" works differently from whoever updated this file before you.)

**Review by severinbeauvais — APPROVED** [review]
LGTM but please wait for a review from one of the code owners.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2569-i3s5egvr.web.app

**Review by seeker25 — APPROVED** [review]

**Review by JazzarKarim — APPROVED** [review]

**Pull request merged**
